ols/app/models/config.py

```diff
@@ -133,7 +133,7 @@ def __init__(self, data: Optional[dict] = None):
         elif self.type == "in-memory":
             if "in-memory" not in data:
                 raise InvalidConfigurationError("in-memory config is missing")
-            self.memory = MemoryConfig(data.get("memory", None))
+            self.memory = MemoryConfig(data.get("in-memory", None))
```

Comment:
make it consistent with line 134 `if "in-memory" not in data`

Reply:
@bparees I recall we talked about it. Is it correct?^

Reply:
we talked about it.  my recommendation was to use `memory` consistently, and not use `in-memory`

Reply:
cc @syedriko since this applies to the config.yaml format/loading also.